### ISSUE TITLE
Add Str::in() method

### ISSUE THREAD

**Issue by PHLAK** [PHLAK/Twine#20]
Determines if the string can be found within another string.

```php
$string = new Twine\Str('pink');

$string->in('john pinkerton'); // Returns true
```

**Comment by thiagormagalhaes** [comment]
I'll work on it

**Comment by aman95** [comment]
Hi @PHLAK 
I've raised a pull request for this issue. Please have a look.
Thanks.

**Comment by PHLAK** [comment]
Resolved by PR #23
